### DISCUSSION TITLE
Does anyone still maintain this repository?

### DISCUSSION THREAD

**Discussion by rtommy** [TomSchimansky/CustomTkinter#2618] [General]
Please allow me to ask....

Based on the lack of [commits](https://github.com/TomSchimansky/CustomTkinter/commits/master/) since January 2024 and the number of open [issues](https://github.com/TomSchimansky/CustomTkinter/issues), may I ask if someone still maintains this repository and actually would improve/correct the source code time to time?

Thank you!

**Comment by dipeshSam** [comment]
@rtommy The following thread might be helpful for you: https://github.com/TomSchimansky/CustomTkinter/discussions/2343

What do you think about above discussion?

Regards.

**Reply by rtommy** [reply]
Thanks!!!!
